### PR TITLE
MAGEDOC-3872-add-import-option-sorting-info

### PR DESCRIPTION
**Pull request by ccondra**
## Purpose of this pull request

This PR explains how to reorder the items listed in a bundle product. I updated the Bundle Product topic to show how to use drag & drop from the Admin, and created a new topic that explains how to add a position parameter to reorder bundle item data in an exported CSV file.

Staging:
- Open Source: https://docs.magedevteam.com/301/m2/ce/user_guide/
- Commerce: https://docs.magedevteam.com/301/m2/ee/user_guide/
- B2B: https://docs.magedevteam.com/301/m2/b2b/user_guide/

## Affected documentation pages

1. **Bundle Product**  (catalog/product-create-bundle.md)
Staging:
https://docs.magedevteam.com/301/m2/ce/user_guide/catalog/product-create-bundle.html
  	A. In Step 5 #5, added instructions to change the order of bundle items and new screenshot.
	    (images/images/product-bundle-items-move.png)
  	B. In Step 5, #6, added instructions to change the order of bundle sections with new screenshot. 
    	(images/images/product-bundle-section-move.png)
	
2. **Importing Bundle Products** (system/data-transfer-bundle-products.md)

- Created new topic that shows how to change the order of items during import. 
Staging:
	https://docs.magedevteam.com/301/m2/ce/user_guide/system/data-transfer-bundle-products.html
	images/images/product-bundle-values-export-data.png
	images/images/product-bundle-values-position-parameter.pn

3. **Data Transfer Examples** (system/data-transfer-examples.md)
- Added Bundle Product link
Staging:
https://docs.magedevteam.com/301/m2/ce/user_guide/system/data-transfer-examples.html

4. Updated **TOCS** (ce/ee/b2b)  - also fixed  "currency" typo in tax section.

## Affected Magento editions

- [ x ] Open Source
- [ x ] Commerce
- [ x  ] B2B

See Contribution guidelines (https://github.com/magento/merchdocs/blob/master/.github/CONTRIBUTING.md) and wiki (https://github.com/magento/merchdocs/wiki) for more information.

### CODE REVIEW

**Comment by ccondra** [comment]
running tests

File: src/system/data-transfer-bundle-products.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: Importing Bundle Products
+---
+
+A bundle product presents a selection of items, and allows customers to choose the ones they want to purchase. All items that make up a bundle already exist in the catalog as either simple or virtual products. Bundle products are usually created and updated from the Admin. However, you can also import data to create a new bundle product, or export existing bundle products, edit the data, and import them back into the catalog. The Sprite Yoga Companion Kit is a bundle product in the sample data that is used in the following examples.
```
Comment (hickskenh) [inline src/system/data-transfer-bundle-products.md]:
```suggestion
A bundle product presents a selection of items and allows customers to choose the ones they want to purchase. All items that make up a bundle already exist in the catalog as either [Simple Products]({% link catalog/product-create-simple.md %}) or [Virtual Products]({% link catalog/product-create-virtual.md %}). Bundle products are usually created and updated from the Admin. However, you can also import data to create a new bundle product or you can export existing bundle products, edit the data, and import them back into the catalog. The Sprite Yoga Companion Kit is a bundle product in the sample data that is used in the following examples.
```

File: src/system/data-transfer-bundle-products.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- Default option indicator 
+- Default quantity
+- Price type
+- Quantity editable indicator
+
+**Step 1: Export the Bundle Product**
+
+In this step, the Sprite Yoga Companion Kit is exported as a ([CSV]({% link system/data-csv.md %})) file. You can use any other bundle product that you have in your catalog.
+
+1.  From the Admin, go to **System** > Data Transfer > **Export**.
```
Comment (hickskenh) [inline src/system/data-transfer-bundle-products.md]:
```suggestion
1.  On the _Admin_ sidebar, go to **System** > Data Transfer > **Export**.  
```

File: src/system/data-transfer-bundle-products.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+**Step 1: Export the Bundle Product**
+
+In this step, the Sprite Yoga Companion Kit is exported as a ([CSV]({% link system/data-csv.md %})) file. You can use any other bundle product that you have in your catalog.
+
+1.  From the Admin, go to **System** > Data Transfer > **Export**.  
+
+1.  Under **Export Settings**, set **Entity Type** to `Products`.
+
+1.  In the list of product attributes, scroll down to **SKU**. Then in the input box, enter the SKU of the bundle product that you want to export. The SKU is 24-WG080 for the product in this example.
```
Comment (hickskenh) [inline src/system/data-transfer-bundle-products.md]:
```suggestion
1.  In the list of product attributes, scroll down to **SKU** and enter the SKU of the bundle product that you want to export. The SKU is `24-WG080` for the product in this example. 
```

File: src/system/data-transfer-bundle-products.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+In this step, the Sprite Yoga Companion Kit is exported as a ([CSV]({% link system/data-csv.md %})) file. You can use any other bundle product that you have in your catalog.
+
+1.  From the Admin, go to **System** > Data Transfer > **Export**.  
+
+1.  Under **Export Settings**, set **Entity Type** to `Products`.
+
+1.  In the list of product attributes, scroll down to **SKU**. Then in the input box, enter the SKU of the bundle product that you want to export. The SKU is 24-WG080 for the product in this example. 
+
+1.  Scroll down to the bottom of the section, and click <span class="btn">Continue</span>.
```
Comment (hickskenh) [inline src/system/data-transfer-bundle-products.md]:
```suggestion
1.  Scroll down to the bottom of the section and click <span class="btn">Continue</span>.
```

File: src/system/data-transfer-bundle-products.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+1.  In the **Action** column of the File name grid, click **Select**, and choose `Download`.
```
Comment (hickskenh) [inline src/system/data-transfer-bundle-products.md]:
```suggestion
1.  In the **Action** column of the _File name_ grid, click **Select** and choose `Download`. 
```

File: src/system/data-transfer-bundle-products.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+1.  Scroll down to the bottom of the section, and click <span class="btn">Continue</span>.
+
+1.  In the **Action** column of the File name grid, click **Select**, and choose `Download`. 
+
+    The file appears in the download location used by your browser.
+
+**Step 2: Edit the Data**
+
+1.  Open the downloaded CSV file in a spreadsheet. Scroll to the far right, until you can see the `bundle_values` column. In the `bundle_values` data, each element is separated by comma, and each bundle item is separated from the next with a vertical bar. (The last item does not end with a vertical bar.) Your exported bundle data should look like this:
```
Comment (hickskenh) [inline src/system/data-transfer-bundle-products.md]:
```suggestion
1.  Open the downloaded CSV file in a spreadsheet. Scroll to the far right, until you can see the `bundle_values` column. In the `bundle_values` data, each element is separated by comma, and each bundle item is separated from the next with a vertical bar. (The last item does not end with a vertical bar.) Your exported bundle data should look similar to the following example:
```

File: src/system/data-transfer-bundle-products.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+    ![]({% link images/images/product-bundle-values-export-data.png
+     %}){: .zoom}
+    _Bundle Values_
+
+1.  To make it easier to edit, you can copy the `bundle_values` data, and paste it into a text editor, Then, add a line break after each item, so each item is on a separate line. After editing the data, carefully remove the line breaks and paste the edited data back into the `bundle_values` column.
```
Comment (hickskenh) [inline src/system/data-transfer-bundle-products.md]:
```suggestion
1.  To make it easier to edit, you can copy the `bundle_values` data and paste it into a text editor. Then, add a line break after each item, so each item is on a separate line. After editing the data, carefully remove the line breaks and paste the edited data back into the `bundle_values` column. 
```

File: src/system/data-transfer-bundle-products.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+    ![]({% link images/images/product-bundle-values-position-parameter.png
+    %}){: .zoom}
+    _Position Parameter_
+
+1.  After editing the data, **Save** the CSV file. 
+
+**Step 3: Import the Updated Product**
+
+1.  From the Admin, go to **System** > Data Transfer > **Import**.
```
Comment (hickskenh) [inline src/system/data-transfer-bundle-products.md]:
```suggestion
1.  On the _Admin_ sidebar, go to **System** > _Data Transfer_ > **Import**.
```

File: src/system/data-transfer-bundle-products.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+   
+1.  Under **Import Settings**, set **Entity Type** to `Products`.
+
+1.  Set **Import Behavior** to `Replace`. This option overwrites the previous data for your bundle product, rather than adding your changes as additional items.
+
+1.  Scroll down to the **File to Import** section. Then, click <span class="btn">Choose File</span> and select the CSV file that you edited.
+
+1.  Click <span class="btn">Check Data</span>and wait a few moments for the data to be checked. If the file is valid, click <span class="btn">Import</span>.
```
Comment (hickskenh) [inline src/system/data-transfer-bundle-products.md]:
```suggestion
1.  Click <span class="btn">Check Data</span> and wait a few moments for the data to be checked. If the file is valid, click <span class="btn">Import</span>.
```

File: src/system/data-transfer-bundle-products.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+1. When the process is complete, go to **System** > Tools > **Cache Management**, and click <span class="btn">Flush Cache Storage</span> to ensure that the updated product is immediately available in the storefront.
```
Comment (hickskenh) [inline src/system/data-transfer-bundle-products.md]:
```suggestion
1. When the process is complete, go to **System** > _Tools_ > **Cache Management** and click <span class="btn">Flush Cache Storage</span>. This ensures that the updated product is immediately available in the storefront.
```

File: src/catalog/product-create-bundle.md [hunk 11]
```diff
@@ -170,6 +170,18 @@ _Choose Template_
     _Item Detail_
 
 1. Repeat these steps for each item you want to add to the bundle.
+   
+1. To change the order of items in a bundle section, click the **Move** ( ![Move icon]({% link images/images/btn-drag.png %}){: .Inline} ) icon at the beginning of the row, and drag the item to the new position.
```
Comment (hickskenh) [inline src/catalog/product-create-bundle.md]:
```suggestion
1. To change the order of items in a bundle section, click the **Move** ( ![Move icon]({% link images/images/btn-drag.png %}){: .Inline} ) icon at the beginning of the row and drag the item to the desired ordered position. 
```

File: src/catalog/product-create-bundle.md [hunk 12]
```diff
@@ -170,6 +170,18 @@ _Choose Template_
...
 1. Repeat these steps for each item you want to add to the bundle.
+   
+1. To change the order of items in a bundle section, click the **Move** ( ![Move icon]({% link images/images/btn-drag.png %}){: .Inline} ) icon at the beginning of the row, and drag the item to the new position. 
+   
+    ![]({% link images/images/product-bundle-items-move.png %}){: .zoom}
+    _Change the Order of Bundle Items_
+
+    The order of items can also be changed in the data of an exported bundle product, and then reimported into the catalog. To learn more, see [Importing Bundle Products]({% link system/data-transfer-bundle-products.md %}).
+
+1. To change the order of sections in a bundle product, click the **Move** ( ![Move icon]({% link images/images/btn-drag.png %}){: .Inline} ) icon in the header of the section, and drag the section to the new position. To have a better view of the workspace, collapse each section first, and then drag them into position.
```
Comment (hickskenh) [inline src/catalog/product-create-bundle.md]:
```suggestion
1. To change the order of sections in a bundle product, click the **Move** ( ![Move icon]({% link images/images/btn-drag.png %}){: .Inline} ) icon in the header of the section and drag the section to the new position. To have a better view of the workspace, collapse each section first and then drag them into position. 
```

File: src/system/data-transfer-bundle-products.md (hunk 1 above)
Comment (ccondra) [reply src/system/data-transfer-bundle-products.md]: I think the comma here is correct.

File: src/catalog/product-create-bundle.md (hunk 11 above)
Comment (ccondra) [reply src/catalog/product-create-bundle.md]: I think this is OK as is.

File: src/catalog/product-create-bundle.md (hunk 12 above)
Comment (ccondra) [reply src/catalog/product-create-bundle.md]: I think this is OK as is.

File: src/system/data-transfer-bundle-products.md (hunk 4 above)
Comment (ccondra) [reply src/system/data-transfer-bundle-products.md]: I think the comma here is correct.

**Comment by ccondra** [comment]
running tests

**Comment by ccondra** [comment]
running tests

File: src/catalog/product-create-bundle.md (hunk 11 above)
Comment (ccondra) [reply src/catalog/product-create-bundle.md]: Can't quite bring myself to use the word "desired" in user docs!  (too much longing and craving there!)  I think "drag the item into position" says it all! :-)

**Comment by ccondra** [comment]
running tests

**Pull request merged**